seaborn/categorical.py

```diff
@@ -2664,6 +2673,9 @@ def boxenplot(x=None, y=None, hue=None, data=None, order=None, hue_order=None,
         Proportion of data believed to be outliers. Used in conjunction with
         k_depth to determine the number of percentiles to draw. Defaults to
         0.007 as a proportion of outliers. Should be in range [0, 1].
+    showfliers : bool, optional
+        If True, outliers beyond ``outlier_prop`` are plotted as markers
```

Comment:
I would change this to `If False, suppress the plotting of outliers`. I want to emphasize that hiding the outliers requires an explicit choice. Also, it's not quite right to say "outliers beyond `outlier_prop`" because it's not a threshold on the continuous value but rather a proportion of observations.